lib/ui/search_ol_editions_screen/search_ol_editions_screen.dart

```diff
@@ -17,20 +17,24 @@ class SearchOLEditionsScreen extends StatefulWidget {
     required this.title,
     this.subtitle,
     required this.author,
+    this.description,
```

Comment:
Is this needed? I don't see it being used anywhere in the file.

Reply:
This is used l. 109. I've only found this way to transmit the data.

Reply:
no, removed